docs/index.mdx

```diff
@@ -8,24 +8,80 @@ slug: /
+
+![First Look Fused Workbench](./index-imgs/Workbench-first-look.png) 
+
+Fused Workbench runs your UDF in real-time. You don't need to execute your code, it's always running, you can see it in the screen shot above:
+1. Our UDF
+2. The NYC subway stations from the geodataframe in the map view
+3. We don't have any errors in the Results Page
+
+:::info
+
+You can also write & run any UDF locally, you don't _need_ the Fused Workbench IDE. 
+
+We'll show how to run your UDF in workbench for now, as that's the simplest way to get started
+
+:::
+
+### Share your User Defined Function
+
+One of the most powerful aspects of Fused is that each UDF can be shared, as a HTTP endpoint. 
+
+Here's how to share our `GeoDataFrame` from Workbench into Google Sheets:
 
-There is currently no comprehensive tool for data scientists to perform their job end-to-end. The modern data stack still has gaps in how data is discovered, accessed, and explored. Writing code to load and transform data can be repetitive. Reusing and sharing code across teams and projects presents its challenges. Creating apps to deliver results to stakeholders can be time-consuming and often involves multiple frameworks.
+import ReactPlayer from 'react-player'
 
-Fused simplifies the process of building and sharing powerful end-to-end data analysis workflows. Fused advances UDFs as composable building blocks that can be called by applications via HTTP calls to load and transform data. As such, UDFs can act as virtual datasets, file readers, and tasks - making analytics easier to scale, share, and integrate.
+<ReactPlayer className="video__player" playing={true} muted={true} controls height="100%" url="https://fused-magic.s3.us-west-2.amazonaws.com/workbench-walkthrough-videos/docs_rewrite/share_udf_to_sheets_updated.mp4" width="100%" />
 
-Workbench consists of the following integrated products designed to enhance how you work with data.
 
-- You use [File Explorer](/workbench/file-explorer/) to browse and preview files in cloud storage
-- [UDF Builder](/workbench/udf-builder/) is where you write UDFs to transform data
-- In the [App Builder](/workbench/app-builder/) you create and share apps that interact with UDFs
-- The [UDF Catalog](/workbench/udf-catalog/) is where you share and discover UDFs
+## Next Steps
 
-[Get started](/user-guide/quickstart/) or jump right into the Workbench at [fused.io/workbench](https://www.fused.io/workbench).
+<!-- #TODO: Update links once the docs are updated with new structure-->
+<!-- #TODO #2: We don't have a page dedicated to workbench-->
+This should give you a first taste of what Fused is. For more, check out:
+- Our [quick start example that walks you through a full end-to-end data science example](/user-guide/quickstart)
+- Get to know what [UDF, User Defined Functions, are in more depth](/core-concepts/why)
+- Become more familiar with Workbench, our browser based, no install required, IDE.
```

Comment:
```suggestion
- Become more familiar with the [UDF Builder](/workbench/udf-builder/), the browser based, no install required, IDE
```

Reply:
The series of commas for the interjection seem awkward. How about:
... UDF Builder, our browser-based IDE (no install required)